stdlib/2.7/__builtin__.pyi

```diff
@@ -858,7 +860,7 @@ class file(BinaryIO):
     def __iter__(self) -> Iterator[str]: ...
     def read(self, n: int = ...) -> str: ...
     def __enter__(self) -> BinaryIO: ...
-    def __exit__(self, typ, exc, tb) -> bool: ...
+    def __exit__(self, t: type, exc: Any, tb: Any) -> bool: ...
```

Comment:
The true signature of `__exit__` allows all three args to be optional. Also, the second could be restricted to `BaseException`.